R/mod_marg.R

```diff
@@ -18,18 +18,21 @@
 #' @export
 #' @examples
 #' data(mtcars)
-#' mtcars$gear <- factor(mtcars$gear)
+#' mtcars$gear <- as.character(mtcars$gear)
 #' mod <- glm(vs ~ gear + mpg * disp, data = mtcars, family = 'binomial')
 #' mod_marg2(mod, var_interest = 'gear',
 #'           type = 'levels', at = list(mpg = c(15, 21), disp = c(140, 180)))
 #'
 #' data(margex)
-#' margex$treatment <- factor(margex$treatment)
 #' mod <- glm(outcome ~ treatment + distance, data = margex, family = 'binomial')
 #' mod_marg2(mod, var_interest = 'treatment', type = 'levels', at = NULL)
 #' mod_marg2(mod, var_interest = 'treatment', type = 'effects', at = NULL)
```

Comment:
@anniejw6 I think these examples need to be updated. Treatment here is running as a continuous variable and the effects line fails with an error (which is working as intended I think, but probably not a great example?)